### PR TITLE
tfel: update to 4.0

### PR DESCRIPTION
**Pull request by raedrizqie**

### CODE REVIEW

File: mingw-w64-tfel/PKGBUILD [hunk 1]
```diff
@@ -16,24 +16,24 @@ makedepends=("unzip"
...
 url='https://tfel.sourceforge.io/'
 source=(https://github.com/thelfer/${_realname}/archive/TFEL-${pkgver}.zip)
-sha256sums=('C7B2EB16AF81DC3A61234A88FCC9EF8401ACDE5EC69D107C99644B74E4D1557A')
+sha256sums=('46bfd2b503f07921f7cdef7479295f95a046f135c4332d1b5939c0fc08263b60')
 noextract=("TFEL-${pkgver}.zip") # bsdtar could not unpack this archive
 
-prepare() {  
-  rm -rf "${srcdir}/TFEL-${pkgver}r"
+prepare() {
+  cd "${srcdir}"
```
Comment (Biswa96) [inline mingw-w64-tfel/PKGBUILD:23]: AFAIK, `cd "${srcdir}"` is not required. Functions in PKGBUILD are automatically starts at `${srcdir}`. More info here https://man.archlinux.org/man/PKGBUILD.5

File: mingw-w64-tfel/PKGBUILD [hunk 2]
```diff
@@ -16,24 +16,24 @@ makedepends=("unzip"
...
+  [[ -d "${srcdir}/build-${MSYSTEM}" ]] && rm -rf "${srcdir}/build-${MSYSTEM}"
+  mkdir -p "${srcdir}/build-${MSYSTEM}" && cd "${srcdir}/build-${MSYSTEM}"
 
   MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
     ${MINGW_PREFIX}/bin/cmake \
       -GNinja \
       -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
       -DCMAKE_BUILD_TYPE=Release \
-      -Denable-fortran=ON \
+      -Denable-fortran=OFF \
```
Comment (Biswa96) [inline mingw-w64-tfel/PKGBUILD:36]: Should fortran be disabled for mingw gcc build also? or only for clang?

Comment (raedrizqie) [reply mingw-w64-tfel/PKGBUILD:36]: ah.. yes.. should be enabled for mingw

File: mingw-w64-tfel/PKGBUILD (hunk 1 above)
Comment (raedrizqie) [reply mingw-w64-tfel/PKGBUILD:23]: ok

**Pull request merged**